bundle/manifests/cluster-logging.clusterserviceversion.yaml

```diff
@@ -390,6 +390,16 @@ spec:
         path: outputs[0].lokiStack.authentication.username.key
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:text
+      - description: Name of the key used to get the value from the referenced Secret.
```

Comment:
I would have expected more changes to this file.  Did you do `make generate bundle` ?

Reply:
I did it again, no other changes.    However, I did notice we still have ref to ES and fluent, and a lot of garbage text.